### PR TITLE
documentation(EJ2-63820): Addressed the review comments

### PR DESCRIPTION
**Pull request by Kokila13081999**
documentation(EJ2-63820): Image loading issue resolved

### CODE REVIEW

File: ej2-asp-core-mvc/mention/EJ2_ASP.MVC/customization.md [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
+title: Customization in ##Platform_Name## Mention Component | Syncfusion
 description: Learn here all about customization in Syncfusion ##Platform_Name## Mention component of Syncfusion Essential JS 2 and more.
 platform: ej2-asp-core-mvc
 control: Customization
 publishingplatform: ##Platform_Name##
 documentation: ug
 ---
 
+# Customization in Mention control
+
```
Comment (thangavele) [inline ej2-asp-core-mvc/mention/EJ2_ASP.MVC/customization.md:12]: image
Dont add the control word.

Comment (Kokila13081999) [reply ej2-asp-core-mvc/mention/EJ2_ASP.MVC/customization.md:12]: Okay sir.. Corrected

File: ej2-asp-core-toc.html [hunk 2]
```diff
@@ -1487,7 +1487,7 @@
 		<li><a href="/ej2-asp-core/mention/working-with-data">Working with Data</a></li>
 		<li><a href="/ej2-asp-core/mention/filtering-data">Filtering Data</a></li>
 		<li><a href="/ej2-asp-core/mention/sorting">Sorting</a></li>
-		<li><a href="/ej2-asp-core/mention/template">Template</a></li>
+		<li><a href="/ej2-asp-core/mention/templates">Template</a></li>
```
Comment (thangavele) [inline ej2-asp-core-toc.html:1490]: Use the Templates word within anchor

Comment (Kokila13081999) [reply ej2-asp-core-toc.html:1490]: Hi @thangavele 

Not understand sir. templates already in anchor tag only. How to modify sir?

File: ej2-asp-core-mvc/mention/EJ2_ASP.MVC/working-with-data.md [hunk 3]
```diff
@@ -40,11 +40,11 @@ The Mention has provided support to load an array of primitive data such as stri
...
-![ASP.NET MVC array of simple data](../images/asp-core-mvc-mention-simple-data.png)
+![ASP.NET MVC Mention array of simple data](./images/asp-core-mvc-mention-simple-data.png)
 
-### 2. Array of JSON data
+### Array of JSON data
 
-The Mention can generate its list of items through an array of JSON data. Therefore the appropriate columns should be mapped to the [fields](../api/mention/#fields) property.
+The Mention can generate its list of items through an array of JSON data. Therefore the appropriate columns should be mapped to the [Fields](../api/mention/#fields) property.
 
 In the following example, `Id` column and `Game` column from complex data have been mapped to the `value` field and `text` field, respectively.
```
Comment (thangavele) [inline ej2-asp-core-mvc/mention/EJ2_ASP.MVC/working-with-data.md:49]: "Id" , You have used the "ID"  in model

Comment (thangavele) [reply ej2-asp-core-mvc/mention/EJ2_ASP.MVC/working-with-data.md:49]: "value" and "text" --> it should be capital letter

File: ej2-asp-core-mvc/mention/EJ2_ASP.NETCORE/working-with-data.md [hunk 4]
```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: Data binding in ##Platform_Name## Mention Component
-description: Learn here all about how to bind with local datasource and how to fetch data from remote data using the service in Syncfusion ##Platform_Name## Mention component of Syncfusion Essential JS 2 and more.
+title: Data binding in ##Platform_Name## Mention Component | Syncfusion
```
Comment (thangavele) [inline ej2-asp-core-mvc/mention/EJ2_ASP.NETCORE/working-with-data.md:3]: ##Platform_Name##  Dont use the platform name template, You can use the direct platform name

File: ej2-asp-core-mvc/mention/EJ2_ASP.NETCORE/working-with-data.md [hunk 5]
```diff
@@ -1,14 +1,14 @@
...
+description: Learn here all about Data Binding in Syncfusion ##Platform_Name## Mention component of Syncfusion Essential JS 2 and more.
```
Comment (thangavele) [inline ej2-asp-core-mvc/mention/EJ2_ASP.NETCORE/working-with-data.md:4]: You should use the control word in MVC, Core, TS, JS

File: ej2-asp-core-mvc/mention/EJ2_ASP.NETCORE/templates.md [hunk 6]
```diff
@@ -38,7 +38,7 @@ In the following sample, the selected value is displayed as a combined text of b
 {% endhighlight %}
```
Comment (thangavele) [inline ej2-asp-core-mvc/mention/EJ2_ASP.NETCORE/templates.md:38]: You should use the asp.net core class reference link
https://help.syncfusion.com/cr/aspnetcore-js2/Syncfusion.EJ2.DropDowns.Mention.html#Syncfusion_EJ2_DropDowns_Mention_DisplayTemplate

Comment (thangavele) [reply ej2-asp-core-mvc/mention/EJ2_ASP.NETCORE/templates.md:38]: Correct it in all section

File: ej2-asp-core-mvc/mention/EJ2_ASP.NETCORE/localization.md [hunk 7]
```diff
@@ -1,17 +1,17 @@
...
 platform: ej2-asp-core-mvc
 control: Localization
 publishingplatform: ##Platform_Name##
 documentation: ug
 ---
 
-# Localization
+# Localization in ASP.NET Core Mention
 
 ASP.NET Core Mention component can be localized. Refer to ASP.NET Core [Localization](../../common/EJ2_ASP.NETCORE/localization.md) topic to localize Syncfusion ASP.NET Core components.
```
Comment (thangavele) [inline ej2-asp-core-mvc/mention/EJ2_ASP.NETCORE/localization.md:13]: components- controls

File: ej2-asp-core-mvc/mention/EJ2_ASP.NETCORE/getting-started.md [hunk 8]
```diff
@@ -101,7 +101,7 @@ Now, add the Syncfusion ASP.NET Core Mention tag helper in `~/Pages/Index.cshtml
```
Comment (thangavele) [inline ej2-asp-core-mvc/mention/EJ2_ASP.NETCORE/getting-started.md:101]: Register Syncfusion Script Manager - is it casing correct?

File: ej2-asp-core-mvc/mention/EJ2_ASP.NETCORE/filtering-data.md [hunk 9]
```diff
@@ -24,7 +24,7 @@ The remote request does not fetch the search data until the search key contains
 {% endhighlight %}
```
Comment (thangavele) [inline ej2-asp-core-mvc/mention/EJ2_ASP.NETCORE/filtering-data.md:24]: `zero`, - dont use the wod, default value is 0

**Comment by Kokila13081999** [comment]
rebuild

File: ej2-asp-core-mvc/mention/EJ2_ASP.MVC/working-with-data.md [hunk 10]
```diff
@@ -57,13 +57,13 @@ In the following example, `Id` column and `Game` column from complex data have b
...
+![ASP.NET MVC Mention array of JSON data](./images/mention-json-data.png)
 
-### 3. Array of Complex data
+### Array of complex data
 
-The Mention can generate its list items through an array of complex data. For this, the appropriate columns should be mapped to the [fields](../api/mention/#fields) property.
+The Mention can generate its list items through an array of complex data. For this, the appropriate columns should be mapped to the [Fields](https://help.syncfusion.com/cr/aspnetmvc-js2/Syncfusion.EJ2.DropDowns.Mention.html#Syncfusion_EJ2_DropDowns_Mention_Fields) property.
 
-In the following example, `Code.Id` and `Country.Name` columns from the complex data have been mapped to the `value` and `text` fields respectively.
+In the following example, `Code.ID` and `Country.Name` columns from the complex data have been mapped to the `Value` and `Text` fields respectively.
```
Comment (rajendranr-5483) [inline ej2-asp-core-mvc/mention/EJ2_ASP.MVC/working-with-data.md:66]: is 'Code' class name?

File: ej2-asp-core-mvc/mention/EJ2_ASP.NETCORE/working-with-data.md [hunk 11]
```diff
@@ -68,11 +68,11 @@ In the following example, `Code.Id` and `Country.Name` columns from the complex
 {% endhighlight %}
 {% endtabs %}
 
-![ASP.NET Core array of complex data](../images/asp-core-mvc-mention-complex-data.png)
+![ASP.NET Core Mention array of complex data](./images/mention-complex-data.png)
 
 ## Binding remote data
 
-The Mention supports retrieval of data from remote data services with the help of `DataManager` control. The [Query](../api/mention/#query) property is used to fetch the data from the database and bind it to the Mention control.
+The Mention supports retrieval of data from remote data services with the help of `DataManager` control. The [query](https://help.syncfusion.com/cr/aspnetcore-js2/Syncfusion.EJ2.DropDowns.Mention.html#Syncfusion_EJ2_DropDowns_Mention_Query) property is used to fetch the data from the database and bind it to the Mention control.
```
Comment (rajendranr-5483) [inline ej2-asp-core-mvc/mention/EJ2_ASP.NETCORE/working-with-data.md:75]: property casing 'Query'

Comment (Kokila13081999) [reply ej2-asp-core-mvc/mention/EJ2_ASP.NETCORE/working-with-data.md:75]: In core platform property name should starts with camel case sir.

File: ej2-asp-core-mvc/mention/EJ2_ASP.MVC/working-with-data.md (hunk 10 above)
Comment (Kokila13081999) [reply ej2-asp-core-mvc/mention/EJ2_ASP.MVC/working-with-data.md:66]: Yes sir.

**Review by thangavele — APPROVED** [review]

**Pull request merged**